lib/loading/cubit/preload/preload_cubit.dart

```diff
@@ -23,7 +23,22 @@ class PreloadCubit extends Cubit<PreloadState> {
         () => BackgroundComponent.preloadAssets(images),
       ),
       PreloadPhase('food', () => FoodComponent.preloadAssets(images)),
-      PreloadPhase('unicorns', () => UnicornComponent.preloadAssets(images)),
+      PreloadPhase(
+        'unicorns',
```

Comment:
```suggestion
        'baby unicorns',
```

Do we want to update the phase names to reflect the asset group or do you prefer to keep them all as unicorns?

Reply:
that is used for i18n purposes, updating to a more awesome message